conf/modules.config

```diff
@@ -41,8 +41,8 @@ process {
         publishDir = [
             path: { "${params.outdir}/genome" },
             mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
-            enabled: params.save_reference
+            pattern: "{!versions.yml}",
```

Comment:
This doesn't work for me. See your tools PR

Reply:
yeah, I saw...

Reply:
I am sadness